py3status/modules/volume_status.py

```diff
@@ -363,10 +363,10 @@ def on_click(self, event):
         button = event['button']
         # volume up
         if button == self.button_up:
-            self.backend.volume_up(self.volume_delta)
+            self.backend.volume_up(event['speed'] or self.volume_delta)
         # volume down
         elif button == self.button_down:
-            self.backend.volume_down(self.volume_delta)
+            self.backend.volume_down(event['speed'] or self.volume_delta)
```

Comment:
Fix this, brother.
* Simplying suggestion... Rename `volume_delta` to `delta` to avoid repeat `volume_`.
    ```python
    volume_status = {
        # volume_delta = 5
        delta = 5
    }
    ```
* Swap the conditions. New default `delta` with `None`. Because this is a new stuff, I wonder if we should immediate replace `delta` (and `brightness_delta` in `backlight`) with the accelerated scrolling to see if users does not need this config anymore.... versus always support `delta = None` config.